go-fuzz/main.go

```
@@ -88,5 +99,31 @@ func main() {
 		go slaveMain()
 	}
 
+	if *flagMasterStats != "" {
+		evMux := http.NewServeMux()
+		evMux.HandleFunc("/eventsource", func(w http.ResponseWriter, r *http.Request) {
+			w.Header().Set("Content-Type", "text/event-stream")
+			w.WriteHeader(http.StatusOK)
+			<-writerSet.Add(w)
+		})
+		evMux.HandleFunc("/stats", func(w http.ResponseWriter, r *http.Request) {
+			w.Header().Set("Content-Type", "text/html")
+			w.WriteHeader(http.StatusOK)
+			io.WriteString(w, statsPage)
+		})
+
+		go http.ListenAndServe(*flagMasterStats, evMux)
```

The error return from `http.ListenAndServe()` is silently discarded.